src/js/components/Grommet/grommet.stories.js

```diff
@@ -51,4 +77,5 @@ const GrommetVars = () => (
 storiesOf('Grommet', module)
   .add('Plain', () => <Plain />)
   .add('Theme', () => <Themed />)
-  .add('Vars', () => <GrommetVars />);
+  .add('Vars', () => <GrommetVars />)
+  .add('Background', () => <Background />);
```

Comment:
we are trying to stay away from having all stories on the same file, can you put the story on a separate file?